libs/brackets-server/package.json

```diff
@@ -24,8 +24,8 @@
     "open": "^0.0.5",
     "rimraf": "^2.2.8",
     "send": "^0.9.3",
-    "socket.io": "1.1.0",
-    "socket.io-client": "1.4.0",
+    "socket.io": "2.0.3",
+    "socket.io-client": "2.0.3",
```

Comment:
What's the benefit of this patch ?
were there any known issue caused by old socket.io ?

Reply:
There is no problem, but It has potential problems.